forms/aws/lambda/organisations/organisations.js

```diff
@@ -0,0 +1,230 @@
+const { RDSDataClient, ExecuteStatementCommand } = require("@aws-sdk/client-rds-data");
+const { Client } = require('pg');
+const { v4: uuidv4 } = require('uuid');
+
+const REGION = process.env.REGION;
+
+exports.handler = async function (event) {
+  console.log(event)
+
+  var dbClient;
+  // Connect to either local or AWS db
+  if (process.env.AWS_SAM_LOCAL) {
+    dbClient = new Client();
+    if (process.env.PGHOST && process.env.PGUSER && process.env.PGDATABASE && process.env.PGPASSWORD) {
+      dbClient.connect();
+    } else {
+      return "Missing Environment Variables for DB config"
+    }
+  } else {
+    dbClient = new RDSDataClient({ region: REGION });
+  }
+
+  /*
+    Supported Methods:
+    INSERT:
+      - (Required) organisationName
+    GET:
+      - (Optional) organisationID. Returns all entries if not provided
+    UPDATE:
+      - All params are required for the first iteration of this lambda
+      - (Required) organisationID
+      - (Required) organisationNameEn
+      - (Required) organisationNameFr
+    DELETE:
+      - (Required) organisationID to delete
+  */
+  
+  const method = event.method;
+  let organisationID = event.organisationID ? event.organisationID : null,
+    organisationNameEn = event.organisationNameEn ? event.organisationNameEn : null,
+    organisationNameFr = event.organisationNameFr ? event.organisationNameFr : null;
+  
+  let SQL = "",
+    parameters = [];
+
+  console.log(method);
+  
+  switch (method) {
+    case "INSERT":
+      if (organisationNameEn && organisationNameFr) {
+        // generate new uuid
+        organisationID = uuidv4();
+        SQL = (!process.env.AWS_SAM_LOCAL) ?
+          "INSERT INTO organisations (id, nameen, namefr) VALUES (:organisationID, :organisationNameEn, :organisationNameFr) RETURNING id"
```

Comment:
Not sure about SQL limitations (it's been awhile) but I'm wondering if we should camel-case the column names for easier readability?

Reply:
That would be an easy fix in a migration in the client.
A (very) quick google has said that underscores are preferred to camel case in SQL columns.